jwst/assign_wcs/assign_wcs_step.py

```diff
@@ -57,7 +57,17 @@ def process(self, input):
             for reftype in self.reference_file_types:
                 reffile = self.get_reference_file(input_model, reftype)
                 reference_file_names[reftype] = reffile if reffile else ""
-            result = load_wcs(input_model, reference_file_names)
+
+            # need to get the imaging mode distorion reference file
+            # is there a better way to do this? Can we have crds assign?
+            exptype_trans = {"NRC_GRISM": "NRC_IMAGE", "NIS_WFSS": "NIS_IMAGE"}
```

Comment:
The Nircam rmap will need to be modified to provide a distortion file for this mode.